lock-api/src/main/java/com/palantir/lock/HeldLocksGrant.java

```diff
@@ -106,7 +106,8 @@ public long getExpirationDateMs() {
      * Returns the set of locks which were successfully acquired, as a mapping
      * from descriptor to lock mode.
      */
-    public SortedLockCollection<LockDescriptor> getLocks() {
+    @Override
+    public SortedLockCollection<LockDescriptor> getLockDescriptors() {
```

Comment:
Shouldn't we keep `getLocks` around as it's a public API method? We could just forward it to `getLockDescriptors` (and possibly deprecate it).